bigquery/bigquery_tags_tag_key/main.tf

```diff
@@ -0,0 +1,27 @@
+/**
+* Copyright 2024 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+# [START bigquery_tags_tag_key]
+resource "google_tags_tag_key" "env_tag_key" {
+  parent     = "projects/myproject1-381000"
```

Comment:
Can we reference project like this instead: https://github.com/terraform-google-modules/terraform-docs-samples/blob/3c210c5a52bba6381da35dc2ba75858ae7d3d190/gke/autopilot/tag/main.tf#L32

Reply:
Done! Thanks!